### PR TITLE
Hsdo 939 section header

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Relies on https://github.com/SU-SWS/stanford_paragraph_types/pull/40
- New simple paragraph type that is just a h2 header page separator.

# Needed By (Date)
- End of sprint

# Urgency
- low

# Steps to Test

1. Checkout branch
2. `drush en stanford_paragraph_types_section_header -y`
3. Create a stanford_page and use the `Section Header` paragraph type with some color.
4. output should just be an H2 tag with some background color.

# See Also
- https://github.com/SU-SWS/stanford_paragraph_types/pull/40
- [linky_clicky](https://github.com/SU-SWS/linky_clicky/pull/171)

### CODE REVIEW

File: css/stanford_paragraph_types_p_cards.css [hunk 1]
```diff
@@ -26,3 +104,39 @@
...
+  background-image: url("../img/brushed-alum.png");
+}
+.paragraphs-item-p-cards .pattern-crisp-paper-ruffles .group-card-title {
+  background-image: url("../img/crisp-paper-ruffles.png");
+}
+.paragraphs-item-p-cards .pattern-diagmonds-light .group-card-title {
+  background-image: url("../img/diagmonds-light.png");
+}
+.paragraphs-item-p-cards .pattern-purty-wood .group-card-title {
+  background-image: url("../img/purty-wood.png");
```
Comment (sherakama) [inline css/stanford_paragraph_types_p_cards.css:141]: I sure hope it is purty.

**Comment by jbickar** [comment]
You wood say that.

**Comment by boznik** [comment]
@pookmish , just verifying this header should span full width.

**Comment by pookmish** [comment]
@boznik, it should be simliar to https://sites.stanford.edu/decker-sandbox/my-teaser-cards. the text should wrap but go full width of the 1170 width

**Pull request merged**